uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -574,13 +576,14 @@ define([
 
         this.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes);
         role = this.directiveOptions.userRole || 'staff';
+        this.canManage = this.isRole('manager') || this.isRole('admin');
```

Comment:
Its better to move 
```
role = this.directiveOptions.userRole || 'staff';
this.canManage = this.isRole('manager') || this.isRole('admin');
```
to a private function with a name like setRoles/initRoles